molr-commons/src/main/java/io/molr/commons/domain/RunState.java

```diff
@@ -10,6 +10,7 @@
     NOT_STARTED,
     RUNNING,
     PAUSED,
+    IGNORED,
```

Comment:
Here I am not sure ... Just we said to keep this slim .. I think I would prefer not to have an additional ignore state ... The ignored block will just never be run ... This should be the same as if the user click skip manually.....